src/content/docs/pt-br/guides/integrations-guide/vue.mdx

```diff
@@ -0,0 +1,235 @@
+---
+type: integration
+title: '@astrojs/vue'
+description: Aprenda como usar a integração de framework @astrojs/vue para estender o suporte a componentes em seu projeto Astro.
+githubIntegrationURL: 'https://github.com/withastro/astro/tree/main/packages/integrations/vue/'
+category: renderer
+i18nReady: true
+---
+import PackageManagerTabs from '~/components/tabs/PackageManagerTabs.astro';
+import Since from '~/components/Since.astro';
+
+Essa **[integração Astro][astro-integration]** habilita a renderização no lado do servidor e a hidratação no lado do cliente para seus componentes [Vue 3](https://vuejs.org/).
+
+## Instalação
+
+Astro inclui um comando `astro add` para automatizar a configuração de integrações oficiais. Se preferir, você pode [instalar as integrações manualmente](#instalação-manual) em vez disso.
+
+Para instalar `@astrojs/vue`, execute o seguinte comando no diretório do seu projeto e siga as instruções:
+
+<PackageManagerTabs>
+  <Fragment slot="npm">
+  ```sh
+  npx astro add vue
+  ```
+  </Fragment>
+  <Fragment slot="pnpm">
+  ```sh
+  pnpm astro add vue
+  ```
+  </Fragment>
+  <Fragment slot="yarn">
+  ```sh
+  yarn astro add vue
+  ```
+  </Fragment>
+</PackageManagerTabs>
+
+Se encontrar qualquer problema, [sinta-se à vontade para relatá-los no GitHub](https://github.com/withastro/astro/issues) e tente os passos de instalação manual abaixo.
+
+### Instalação Manual
+
+Primeiro, instale o pacote @astrojs/vue:
```

Comment:
```suggestion
Primeiro, instale o pacote `@astrojs/vue`:
```

Reply:
Thank you for fixing this, @yanthomasdev